src/FileHelper.php

```diff
@@ -155,6 +155,11 @@ public static function normalizePath(string $path): string
      * @param array $options Options for directory remove ({@see clearDirectory()}).
      *
      * @throw RuntimeException when unable to remove directory.
+     *
+     * @psalm-param array{
+     *     traverseSymlinks?:bool,
+     *     filter?:PathMatcherInterface|mixed
```

Comment:
Why need filter for `removeDirectory()`? If use filter, thid directory can be not removed.

Reply:
`clearDirectory()` uses `removeDirectory()`.
I thought that if the directory contains a filtered file, then it needs to be saved. But it would probably be more correct to delete the directory

```
  public static function removeDirectory(string $directory, array $options = []): void
    {
        if (!file_exists($directory)) {
            return;
        }

        /** @psalm-suppress InvalidArrayOffset */
        unset($options['filter']);

        self::clearDirectory($directory, $options);

//        if (self::getFilter($options) !== null && !self::isEmptyDirectory($directory)) {
//            return;
//        }
```